examples/plot_psd_slope_estimation.py

```diff
@@ -76,15 +75,15 @@
 # by ``compute_spect_slope`` because, in the feature function, the linear
 # regression fit is done in the log10-log10 scale.
 intercept, slope, _, _ = compute_spect_slope(sfreq, data, fmin=1., fmax=40.)
-print('The estimated slope (respectively intercept) is: %1.2f (resp. %1.3e)' %
-      (slope, intercept))
+print('The estimated slope is a = %1.2f and the estimated intercept is '
+      'b = %1.3e' % (slope, intercept))
 
-# Plot the PSD together with the ``b / (f ** a)`` curve (estimated decay of
-# the PSD with frequency).
+# Plot the PSD together with the ``b + a * f`` straight line (estimated decay
+# of the PSD with frequency in the log10-log10 scale).
 plt.figure()
 plt.semilogx(freqs, np.log10(psd), '-b', lw=2, label='PSD')
 plt.semilogx(freqs, intercept + slope * np.log10(freqs),
-             '-r', lw=2, label='b / (f ** a)')
+             '-r', lw=2, label='b + a * f')
```

Comment:
are you sure?

Reply:
Data to be plotted on the y axis (no log applied on this axis with `semilogx`) is `intercept + slope * np.log10(freqs)`. That's why I changed the legend to `b + a * f`.

Reply:
ok